moveos/moveos-stdlib/src/natives/moveos_stdlib/object.rs

```diff
@@ -153,6 +153,23 @@ pub fn native_share_object(
     Ok(NativeResult::ok(cost, smallvec![]))
 }
 
+// ObjectBox keep same layout with Object struct in Move
+pub(crate) struct ObjectBox {
+    pub id: AccountAddress,
```

Comment:
If the account address is used directly for the object id, it means that if the recipient address passed in when transferring is the object id, because it cannot be verified, the owner will become the object id and the object will never be accessible again.

Reply:
It is like transferring Token to a no exists account.